packages/snap-toolbox/src/DomTargeter/DomTargeter.ts

```diff
@@ -95,6 +97,11 @@ export class DomTargeter {
 			if (target.inject) {
 				try {
 					const injectedElem = this.inject(elem, target);
+
+					if (target.inject.action !== 'replace') {
+						targetedElems = targetedElems.concat(elem);
```

Comment:
Don't we want to add the `injectedElem` to the `targetedElems` array?

Reply:
yeeeep